src/ngAnimate/animateCss.js

```diff
@@ -742,6 +742,9 @@ var $AnimateCssProvider = ['$animateProvider', function($animateProvider) {
         applyAnimationClasses(element, options);
         applyAnimationStyles(element, options);
 
+        // we increment the counter so that if the same element is animated
+        // then the existing cache will not apply to it
+
```

Comment:
I don't understand the context of this comment. Where is the counter that is being incremented?